spec/factories/follower.rb

```diff
@@ -0,0 +1,12 @@
+require './lib/pivotal-tracker/follower'
+
+FactoryGirl.define do
+  factory :follower, class: PivotalTracker::Follower do
+    sequence(:id) { |n| n }
+    name          { Faker::Name.name }
+    email         { Faker::Internet.email }
+    initials      { name.split.map{|name| name[0] } }
+    username      { Faker::Internet.user_name }
+    kind          'kind'
```

Comment:
All of these `kind` values should be what the API returns...  A good practice is to make the factories look like data you would use in the real world.  Developer users of the gem can look to the factories to determine what the data looks like.